src/op_mode/show_interfaces.py

```diff
@@ -202,12 +202,12 @@ def run_show_intf(ifnames, iftypes, vif, vrrp):
 
 @register('show-brief')
 def run_show_intf_brief(ifnames, iftypes, vif, vrrp):
-    format1 = '%-16s %-33s %-4s %s'
+    format1 = '%-16s %-33s %-9s %-4s %s'
     format2 = '%-16s %s'
 
     print('Codes: S - State, L - Link, u - Up, D - Down, A - Admin Down')
-    print(format1 % ("Interface", "IP Address", "S/L", "Description"))
-    print(format1 % ("---------", "----------", "---", "-----------"))
+    print(format1 % ("Interface", "IP Address","VLAN Mode", "S/L", "Description"))
```

Comment:
Please do not change this - we should discuss the format in a dedicated task.

Reply:
It actually breaks when no VLAN interfaces exist:

```
cpo@LR1.wue3# run show interfaces
Traceback (most recent call last):
  File "/usr/libexec/vyos/op_mode/show_interfaces.py", line 305, in <module>
Codes: S - State, L - Link, u - Up, D - Down, A - Admin Down
Interface        IP Address                        VLAN Mode S/L  Description
---------        ----------                        --------- ---  -----------
dum0             172.18.254.201/32                 Disabled  u/u
eth0             -                                 Disabled  u/u
eth0.201         172.18.201.10/24                  Disabled  u/u
    args.vrrp
  File "/usr/libexec/vyos/op_mode/show_interfaces.py", line 223, in run_show_intf_brief
    mode = [_ for _ in interface.get_vlan_mode_bind_bridge()]  or ['Disabled', ]
  File "/usr/lib/python3/dist-packages/vyos/ifconfig/interface.py", line 727, in get_vlan_mode_bind_bridge
    with open(f'/sys/class/net/{bridge_interface_name}/bridge/vlan_filtering', 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/sys/class/net/br0/bridge/vlan_filtering'
```

Reply:
I will cancel it in this implementation and archive the correction of the shoe interfaces locally for later use. But I don't understand why I can't find the VLAN filtering path of br0. Strange? This switch should be available for every bridge device and does pass the test on the local virtual machine